sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/_base.py

```diff
@@ -433,3 +438,32 @@ def _format_storage_telemetry_item(item: TelemetryItem) -> TelemetryItem:
                     item.data.base_data = base_type.from_dict(item.data.base_data.additional_properties)  # type: ignore
                     item.data.base_data.additional_properties = None  # type: ignore
     return item
+
+# mypy: disable-error-code="union-attr"
+def _get_authentication_credential(**kwargs: Any) -> Optional[ManagedIdentityCredential]:
+    if "credential" in kwargs:
+        return kwargs.get("credential")
+    try:
+        if _APPLICATIONINSIGHTS_AUTHENTICATION_STRING in os.environ:
+            auth_string = os.getenv(_APPLICATIONINSIGHTS_AUTHENTICATION_STRING, "")
+            kv_pairs = auth_string.split(";")
+            auth_string_d = {}
+            for kv_pair in kv_pairs:
+                key_value = kv_pair.split("=")
+                if len(key_value) == 2:
+                    (key, value) = key_value
+                    auth_string_d[key.lower()] = value
+                else:
+                    logger.error("Invalid APPLICATIONINSIGHTS_AUTHENTICATION_STRING format: %s", auth_string)
+            if "authorization" in auth_string_d and auth_string_d["authorization"] == "AAD":
+                if "clientid" in auth_string_d:
+                    logger.info("ClientId found, trying to authenticate using Managed Identity.")
```

Comment:
I don't believe we log `info` level logs in our SDK besides from diagnostic logging, is there a reason why we need this information?

Reply:
Not really. I can remove. Or maybe I'll make them diagnostic logs under an attach check.

Reply:
Since the exporter does not have access to diagnostic laws, I just removed.